From cc5212a71906a3d38956beed2f95554864571e59 Mon Sep 17 00:00:00 2001
From: "bo.yang" <bo.yang@amlogic.com>
Date: Tue, 9 Dec 2014 10:52:33 +0800
Subject: [PATCH 5167/5965] Revert "g9tv: add workround for clock to support
 rev a&b"

This reverts commit a4cbb381d82cc77ea10543e4fb2dba69ca26166a.
---
 arch/arm/mach-mesong9tv/clock.c | 16 +++-------------
 1 file changed, 3 insertions(+), 13 deletions(-)

diff --git a/arch/arm/mach-mesong9tv/clock.c b/arch/arm/mach-mesong9tv/clock.c
index b3853ae25aa8..01ad9300e8a9 100755
--- a/arch/arm/mach-mesong9tv/clock.c
+++ b/arch/arm/mach-mesong9tv/clock.c
@@ -54,9 +54,7 @@ static DEFINE_MUTEX(clock_ops_lock);
 #define SYS_PLL_TABLE_MIN		24000000
 #define SYS_PLL_TABLE_MAX		2112000000
 
-#define CPU_FREQ_LIMIT			1200000000
-#define CPU_FREQ_LIMIT_REVA		1488000000
-
+#define CPU_FREQ_LIMIT			1488000000
 
 struct sys_pll_s {
 	unsigned int freq;
@@ -984,16 +982,11 @@ static int clk_set_rate_a9(struct clk *clk, unsigned long rate)
 {
 	int ret;
 	unsigned long irq_flags;
-	unsigned int cpu_freq_limit = CPU_FREQ_LIMIT;
 
 	//printk("clk_set_rate_a9() clk: %d\n",rate);
 
-	if (IS_MESON_MG9TV_CPU_REVA) {
-		cpu_freq_limit = CPU_FREQ_LIMIT_REVA;
-	}
-
-	if(rate > cpu_freq_limit)
-		rate = cpu_freq_limit;
+	if(rate > CPU_FREQ_LIMIT)
+		rate = CPU_FREQ_LIMIT;
 
 	irq_flags = arch_local_irq_save();
 	preempt_disable();
@@ -1755,9 +1748,6 @@ static int __init meson_clock_init(void)
 		printk("Error: clk81 not be selected at fixed_pll.\n");
 	}
 
-	if (IS_MESON_MG9TV_CPU_REVA) {
-		setup_a9_clk_max = CPU_FREQ_LIMIT_REVA;
-	}
 	// Add CPU clock
 	CLK_DEFINE(a9_clk, pll_sys, -1, clk_set_rate_a9, clk_get_rate_a9, NULL, NULL, NULL);
 	clk_a9_clk.min = setup_a9_clk_min;
-- 
2.19.0

